.vsts-ci.yml

```diff
@@ -345,17 +351,48 @@ stages:
 
   - template: /eng/common/templates/jobs/source-build.yml
 
-  - template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml
+- stage: Build_Linux
+  displayName: Build Linux (for Source-Build)
+  dependsOn: []
+  jobs:
+  - template: /eng/build.yml
+    parameters:
+      agentOs: Linux
+      jobName: Build_LinuxPortable_Release_x64
+      buildConfiguration: Release
+      buildArchitecture: x64
+      linuxPortable: true
+      runTests: false
+  - template: /eng/build.yml
+    parameters:
+      agentOs: Linux
+      jobName: Build_Arm64_Release
+      buildConfiguration: Release
+      buildArchitecture: arm64
+      runtimeIdentifier: 'linux-arm64'
+      linuxPortable: true
+      runTests: false
 
-  - ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
-    - template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml
+- ${{ if or(ne(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+  - stage: Synchronize_VMR
+    displayName: Synchronize VMR
+    dependsOn: []
+    jobs:
+    - template: /eng/pipelines/templates/jobs/vmr-synchronization.yml
       parameters:
-        dependsOn: Source_Build_Create_Tarball
+        vmrBranch: ${{ parameters.vmrBranch }}
+        noPush: ${{ parameters.noPush }}
 
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - template: /eng/pipelines/templates/stages/vmr-source-build.yml
```

Comment:
I think this should be conditioned to only run for PRs.  We should not be running the tarball CI in other scenarios especially the full matrix.  This is a waste of compute.

Reply:
Yeah, this simplified things greatly down the pipeline - we don't need to pass the VMR SHA through the pipeline variables then